extensions/spring-xd-extension-batch/src/main/java/org/springframework/batch/step/tasklet/x/AbstractProcessBuilderTasklet.java

```diff
@@ -236,23 +236,23 @@ public ExitStatus afterStep(StepExecution stepExecution) {
 	protected abstract String getCommandDescription();
 
 	protected String createClassPath(Class taskletClass) {
-		URLClassLoader thisClassLoader;
```

Comment:
Now I'm being picky... But wouldn't this affect only the SqoopTasklet and not the Spark Tasklet?